abuild-fetch.c

```diff
@@ -124,25 +124,33 @@ int fetch(char *url, const char *destdir)
 
 	childpid = fork();
 	if (childpid < 0 )
-		err(1, "fork");
+		err(200, "fork");
 
 	if (childpid == 0) {
 		execvp(curlcmd.argv[0], curlcmd.argv);
 		printf("Using wget\n");
 		execvp(wgetcmd.argv[0], wgetcmd.argv);
 		warn("%s", wgetcmd.argv[0]);
 		unlink(lockfile);
-		_exit(1);
+		_exit(201);
 	}
+
+	/* wait for curl/wget and get the exit code */
 	wait(&status);
-	rename(partfile, outfile);
+	if(WIFEXITED(status))
```

Comment:
Please follow the code-style used in this project – space between `if` and parenthesis.